lib/node_modules/@stdlib/stats/base/snanmeanpn/benchmark/benchmark.js

```diff
@@ -21,16 +21,30 @@
 // MODULES //
 
 var bench = require( '@stdlib/bench' );
-var randu = require( '@stdlib/random/base/randu' );
 var isnan = require( '@stdlib/math/base/assert/is-nan' );
 var pow = require( '@stdlib/math/base/special/pow' );
-var Float32Array = require( '@stdlib/array/float32' );
+var uniform = require( '@stdlib/random/base/uniform' );
+var bernoulli = require( '@stdlib/random/base/bernoulli' );
+var filledarrayBy = require( '@stdlib/array/filled-by' );
 var pkg = require( './../package.json' ).name;
 var snanmeanpn = require( './../lib/snanmeanpn.js' );
 
 
 // FUNCTIONS //
 
+/**
+* Returns a random value or `NaN`.
+*
+* @private
+* @returns {number} random number or `NaN`
+*/
+function rand() {
+	if ( bernoulli( 0.2 ) ) {
```

Comment:
```suggestion
	if ( bernoulli( 0.8 ) < 1 ) {
```
can you please do this instead? this is the convention that we're going to follow from now on.

Reply:
same comment for all js benchmarks